### PR TITLE
Class basic syntax

### PR DESCRIPTION
**Pull request by Benyaak**
Finished the class basic syntax lesson, I hope everything is okay. Waiting for a review/translation suggestions. Thanks a lot.

### CODE REVIEW

File: 1-js/09-classes/01-class/1-rewrite-to-class/task.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/1-rewrite-to-class/task.md]: Missing translation: "Rewrite to class"

Comment (Benyaak) [reply 1-js/09-classes/01-class/1-rewrite-to-class/task.md]: Solved & Pushed

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md]: Missing translation : "...a class is an extensible program..." -> "o clasa este un program extensibil"

Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md]: Typo : "variabileleor" -> "variabilelor"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md]: Grammar suggestion : "despre ce este o clasa" -> "ce este defapt o clasa"

Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md]: Faulty translation: "O sa incepem sa intelegem multe aspecte complexe ale acesteia" -> "Asta ne va ajuta sa-i intelegem multe aspecte complexe"

**Comment by Benyaak** [comment]
Solved all suggestion /done

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:98]: Grammar suggestion : "a-l introspecta" -> "a-l analiza"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:121]: Grammar suggestion : "....sintaxa implicita...." -> "...sintaxa explicita..."

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:131]: Grammar suggestion : "functie prototip" -> "prototip de functie"

Typo : "mode implicit" -> "mod implicit"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:143]: Typo : "intreadevar" -> "intr-adevar"

Comment (lighthousand) [reply 1-js/09-classes/01-class/article.md:143]: Typo : "pentru a definii" -> "pentru a defini"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:147]: Typo : "creadta" -> "creata"

Comment (lighthousand) [reply 1-js/09-classes/01-class/article.md:147]: Typo : "altfel" -> "astfel"

Comment (lighthousand) [reply 1-js/09-classes/01-class/article.md:147]: Grammar suggestion : "cu crearea manuala" -> "ca și crearea manuală"

Comment (lighthousand) [reply 1-js/09-classes/01-class/article.md:147]: Forgot period at the end of the sentence.

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:160]: Typo : "engine-ul JavaScript" -> "engine-urile Javascript"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:173]: Grammar suggestion : "este foarte bine" -> "este bine"

Comment (lighthousand) [reply 1-js/09-classes/01-class/article.md:173]: Typo : "daca" -> "dacă"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:177]: Remove the 2nd sentence

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:179]: Grammar suggestion : "In plus" -> "In afara de asta"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:183]: Grammar suggestion : "in interiorul alte expresii" -> "in interiorul altor expresii"

Comment (lighthousand) [reply 1-js/09-classes/01-class/article.md:183]: Grammar suggestion : "returnare" -> "returnate"
Grammar suggestion : "alocate" -> "atribuite"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:197]: Typo : "Daca" -> "Dacă"

Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:197]: Grammar suggestion : "este vizibil" -> "este vizibilă"

**Comment by lighthousand** [comment]
@Benyaak, you're not a romanian native speaker, am i right ? :)

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:226]: Grammar suggestion : "Crează" -> "Creează"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:235]: Grammar suggestion : "getteri/setter" -> "getteri/setteri"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:271]: Grammar suggestion : " crează" -> "creează"

Comment (lighthousand) [reply 1-js/09-classes/01-class/article.md:271]: Grammar suggestion : "setter" -> "setteri"

Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:271]: Grammar suggestion : "exact așa" -> "astfel"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:286]: Grammar suggestion : "paranteze" -> "paranteze pătrate"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:302]: Grammar suggestion : "o precedăm" -> "o prefixăm"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:307]: Typo : "Proprietațile" -> "Proprietățile"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:310]: Grammar suggestion : "În exemplul de mai sus" -> "Exemplul de mai sus"

Comment (lighthousand) [reply 1-js/09-classes/01-class/article.md:310]: Typo : "adăugam" -> "adăugăm"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:326]: Grammar suggestion : "creată de" -> "creeată de"

Comment (lighthousand) [reply 1-js/09-classes/01-class/article.md:326]: There's a part that has not been translated : "it's the property of the object itself."

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:328]: Grammar suggestion : "Rezumat"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:330]: Grammar suggestion : "a unei clase" -> "a clasei"

File: 1-js/09-classes/01-class/article.md
Comment (lighthousand) [inline 1-js/09-classes/01-class/article.md:352]: Grammar suggestion : "alte caracteristici ale acestora" -> "alte caracteristici"

Comment (lighthousand) [reply 1-js/09-classes/01-class/article.md:352]: Grammar suggestion : "printre care o să învătăm despre " -> "printre care moștenire și alte caracteristici"

**Review by lighthousand — DISMISSED** [review]
Overall, good job !

**Review by bogdanbacosca — APPROVED** [review]

**Comment by bogdanbacosca** [comment]
@iliakan I can't merge this one. It's quite stale, long overdue. If you merge, I can take care to fix it.

**Review by bogdanbacosca — APPROVED** [review]

**Comment by iliakan** [comment]
You can't merge because of conflicts. Can you fix the conflicts (know what it is?)

**Comment by bogdanbacosca** [comment]
Yep but GitHub wouldn't let me edit the files. I think this person unticked the "let maintainers edit code" box

**Comment by iliakan** [comment]
Maybe check it out locally, merge/edit and then close this PR?

**Comment by bogdanbacosca** [comment]
I think he may have deleted his `branch` - `checkout` to `Benyaak:master` just creates a brand *new* branch. Also I cannot see his pull request in the GitHub desktop app, the other one shows up but not his.

Not sure exactly, I can copy/paste his code and modify it in a new `branch` because it seems like there are not so many mistakes, but not sure if it's "legal" or fair to do that. Translating one article is not that much work from my experience so far and I wouldn't mind to let go of it also.

Is there another way?

**Comment by iliakan** [comment]
```
 git fetch -fu origin refs/pull/40/head:pr/40
 git checkout pr/40
```

**Comment by bogdanbacosca** [comment]
``` console
me@os:~/Documents/GitHub/ro.javascript.info$ git fetch -fu origin refs/pull/40/head:pr/40
fatal: couldn't find remote ref refs/pull/40/head

```

**Comment by iliakan** [comment]
From this repo:
```
➜  ro.javascript.info git:(master) git fetch -fu origin refs/pull/40/head:pr/40
remote: Enumerating objects: 27, done.
remote: Counting objects: 100% (11/11), done.
remote: Total 27 (delta 11), reused 11 (delta 11), pack-reused 16
Unpacking objects: 100% (27/27), 6.31 KiB | 646.00 KiB/s, done.
From github.com:javascript-tutorial/ro.javascript.info
 * [new ref]           refs/pull/40/head -> pr/40
```

**Comment by iliakan** [comment]
I made a new branch `tmp-40` with this PR and pushed it.

**Comment by iliakan** [comment]
See this PR https://github.com/javascript-tutorial/ro.javascript.info/pull/146

**Comment by bogdanbacosca** [comment]
thanks a lot 🙏️

**Pull request closed without merging**
